CHANGELOG.asciidoc

```diff
@@ -652,13 +652,6 @@ https://github.com/elastic/beats/compare/v1.2.0...v5.0.0-alpha1[View commits]
 - Add `event_id`, `level`, and `provider` configuration options for filtering events {pull}1218[1218]
 - Add `include_xml` configuration option for including the raw XML with the event {pull}1218[1218]
 
-==== Known issues
```

Comment:
@tsg Keep them in would mean we have a type "Known issues" in the script. I think it is a good idea as it will allow us to put the known issues into the yml file but use the script to put them where we need it.